Source/WebKit/Shared/Cocoa/CoreIPCDictionary.mm

```diff
@@ -55,8 +55,11 @@
 {
     createNSDictionaryIfNeeded();
     id object = [m_nsDictionary.get() objectForKey:(NSString *)key];
-    if (!object)
-        return false;
+    if (!object) {
+        // Many objects have a required key that sometimes has a missing value, which is okay.
```

Comment:
doesn't that mean it's not a required key?

Reply:
It's ill formed for the key to be missing.
But the key can be null.